### DISCUSSION TITLE
Group by author when books aren't in a series?

### DISCUSSION THREAD

**Discussion by BMStroh** [Kareadita/Kavita#3092] [Ideas]
### Idea Description

This may be a matter of just figuring out the filename export format from Calbre, but it might make for a cleaner library if non-series books were grouped by author, using the publication date as the series index to sort. 

Anyone else think this might be good to document, at least as an option?

### Idea Category

User Experience

### Duration of Using Kavita

3 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by KeePach** [comment]
I had this same idea. A way to group books that aren't series by author

**Comment by pdf** [comment]
I don't know, there's already an Authors view, so users wanting to browse by author can do so already. For me, I'd just like to see standalone novels not added to any series grouping - I often want to browse series specifically, but having all the standalone novels mixed in there makes having a series view largely meaningless, and whilst putting novels into author "series" would reduce the clutter somewhat, it would still add a lot of noise to the series list. Ideally the dashboard and library views would show both standalone books and series in this scenario.

The current approach probably works okay for comics/manga, since they're almost all series (and I guess that's where the data model came from), but for regular books it's pretty painful, and in evaluating a move from calibre-web to Kavita, not having a way to browse only series is a show-stopper for me.
